packages/manager/src/features/ObjectStorage/BucketLanding/BucketRateLimitTable.test.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+
+import { renderWithTheme } from 'src/utilities/testHelpers';
+
+import { BucketRateLimitTable } from './BucketRateLimitTable';
+
+describe('BucketRateLimitTable', () => {
+  it('should render a BucketRateLimitTable', () => {
+    const { getAllByText, getByText } = renderWithTheme(
+      <BucketRateLimitTable endpointType="E1" />
```

Comment:
Table will never show for anything other than E2 or E3
```suggestion
      <BucketRateLimitTable endpointType="E2" />
```